.travis.yml

```diff
@@ -19,6 +19,7 @@ r_github_packages:
   - jeroenooms/curl
   - klutometis/roxygen
   - jimhester/covr
+  - yihui/mime
   - ropensci/geojsonio
```

Comment:
We now also require `mime` version >= 4.0 which has been published to CRAN
https://cran.rstudio.com/web/packages/mime/index.html